webapp/src/app/shared/phaser/tilemap/tile-drawer.ts

```diff
@@ -31,6 +31,7 @@ export class TileDrawer extends BaseObject {
 	private fillKey!: Phaser.Input.Keyboard.Key;
 	private lastDraw: Point = {x: -1, y: -1};
 	
+	private panKey!: Phaser.Input.Keyboard.Key;
```

Comment:
the pan key is currently defined 3 times in 3 different files. Ideally it should only be defined in map-pan and update some global state so the other functions can access it. E.g. adding a  variable `panning: boolean` to the class `Globals`

Reply:
@Vegita2 Glad you said something,  had the same thought about duplicated code.  I'll give updating this to be globals a shot and push an update when ready.

Reply:
@Vegita2 okay, added a new global, set in `map-pan`,  works good